src/index.css

```diff
@@ -1,13 +1,184 @@
 :root {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+    margin: 0;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto",
+        "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans",
+        "Helvetica Neue", sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    background-color: rgb(0, 0, 0);
+    color: #fff;
+    overflow-x: hidden;
+}
+
+* {
+    margin: 0;
+    box-sizing: border-box;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+    font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+        monospace;
+    margin: 0;
+}
+
+a {
+    text-decoration: none;
+}
+
+/*NAVBAR*/
+.nav-bar {
+    background-color: black;
+    width: 100%;
+    padding: 2rem;
+    margin: 0;
+    text-decoration: none;
+    font-size: 16px;
+    text-align: center;
+    border-bottom: #fff 1px solid;
+}
+
+.nav-bar ul {
+    background-color: black;
+    display: flex;
+    justify-content: space-around;
+    margin: 0;
+    padding: 0;
+}
+
+.nav-bar ul li {
+    list-style-type: none;
+}
+
+.nav-bar h1 {
+    color: #fff;
+}
+
+a.active {
+    text-decoration: #fff  3px underline overline;
+    text-underline-offset: 6px;
+}
+
+/*MOVIELIST*/
+.movieList-section {
+    margin: 3rem 2rem; 
+    display: grid;
+    grid-gap: 1rem;
+    grid-template-columns: repeat(auto-fill, minmax(15rem, 1fr));
+}
+/*MOVIE CARD*/
+.movie-card {
+    background-color: black;
+    border-radius: 1rem;
+    color: white;
+    height: 100%;
+    border: transparent 1px solid;
+}
+
+.image-overlay {
+    position: relative;
+    width: 100%;
+}
+
+.image-overlay::before {
+    content: "";
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    background: linear-gradient(to bottom,rgba(0, 255, 242, 0), rgba(0, 0, 0, 0), rgb(0, 0, 0) );
+}
+.movie-card img {
+    width: 100%;
+    border-radius: 1rem 1rem 0rem 0rem;
+}
+
+.movie-card-text-content {
+    padding: 1rem 1rem 2rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem; 
+}
+
+.movie-card-text-content p {
+    color: rgb(180, 180, 180);
+}
+.movie-card:hover {
+    scale: 1.02;
+    border: #fff 1px solid;
+}
+
+/*MOVIE DETAILS SECTION*/
```

Comment:
When I enter this page I thought you didn't have the "upcoming" and "popular" buttons at first as I needed to scroll up in order to see it. It doesn't appear at first for me. Don't know if has to do with a different browser?